### ISSUE TITLE
[Bug] Crash on scrolling down in Newpipe's app sheet

### ISSUE THREAD

**Issue by opusforlife2** [NeoApplications/Neo-Store#640]
**Describe the bug**

The app immediately crashes when reaching the bottom of the Newpipe app sheet.

Note: Crash is on 1.0.2. 1.0.3 needed a hotfix, which hasn't reached F-Droid yet, so I'm waiting on 1.0.4 still.

**To Reproduce**

1. Open Newpipe's app sheet.
2. Scroll to the bottom.

**Expected behavior**

No crash.

**Logs**

```
FATAL EXCEPTION: main
Process: com.machiav3lli.fdroid, PID: 32059
java.lang.IllegalArgumentException: Key "997#1" was already used. If you are using LazyColumn/Row please make sure you provide a unique key for each item.
	at androidx.compose.ui.layout.LayoutNodeSubcompositionsState$Scope.subcompose(SourceFile:162)
	at androidx.compose.foundation.lazy.layout.LazyLayoutMeasureScopeImpl.measure-0kLqBqw(SourceFile:34)
	at androidx.compose.foundation.lazy.LazyListKt$rememberLazyListMeasurePolicy$1$1$measuredItemProvider$1.getAndMeasure(SourceFile:21)
	at androidx.compose.foundation.lazy.LazyListKt$rememberLazyListMeasurePolicy$1$1.invoke(SourceFile:750)
	at androidx.compose.material3.ButtonKt$Button$3$1.invoke(SourceFile:314)
	at androidx.compose.ui.layout.LayoutNodeSubcompositionsState$createMeasurePolicy$1.measure-3p2s80s(SourceFile:78)
	at androidx.compose.ui.node.InnerNodeCoordinator.measure-BRTryo0(SourceFile:38)
	at androidx.compose.foundation.AndroidOverscroll_androidKt$StretchOverscrollNonClippingLayer$1.invoke(SourceFile:14)
	at androidx.compose.ui.layout.LayoutModifierImpl.measure-3p2s80s(SourceFile:8)
	at androidx.compose.ui.node.LayoutModifierNodeCoordinator.measure-BRTryo0(SourceFile:11)
	at androidx.compose.foundation.AndroidOverscroll_androidKt$StretchOverscrollNonClippingLayer$1.invoke(SourceFile:48)
	at androidx.compose.ui.layout.LayoutModifierImpl.measure-3p2s80s(SourceFile:8)
	at androidx.compose.ui.node.LayoutModifierNodeCoordinator.measure-BRTryo0(SourceFile:11)
	at androidx.compose.ui.graphics.SimpleGraphicsLayerModifier.measure-3p2s80s(SourceFile:1)
	at androidx.compose.ui.node.LayoutModifierNodeCoordinator.measure-BRTryo0(SourceFile:11)
	at androidx.compose.foundation.layout.FillNode.measure-3p2s80s(SourceFile:97)
	at androidx.compose.ui.node.LayoutModifierNodeCoordinator.measure-BRTryo0(SourceFile:11)
	at androidx.compose.runtime.Pending$keyMap$2.invoke(SourceFile:3)
	at androidx.compose.runtime.snapshots.SnapshotIdSetKt.observe(SourceFile:60)
	at androidx.compose.runtime.snapshots.SnapshotStateObserver$ObservedScopeMap.observe(SourceFile:48)
	at androidx.compose.runtime.snapshots.SnapshotStateObserver.observeReads(SourceFile:153)
	at androidx.compose.ui.node.OwnerSnapshotObserver.observeReads$ui_release(Unknown Source:2)
	at androidx.compose.ui.node.LayoutNodeLayoutDelegate$MeasurePassDelegate.remeasure-BRTryo0(SourceFile:129)
	at androidx.compose.ui.node.MeasureAndLayoutDelegate.doRemeasure-sdFAvZA(SourceFile:18)
	at androidx.compose.ui.node.MeasureAndLayoutDelegate.measureAndLayout-0kLqBqw(SourceFile:70)
	at androidx.compose.ui.platform.AndroidComposeView.measureAndLayout-0kLqBqw(SourceFile:8)
	at androidx.compose.ui.node.LayoutNode.forceRemeasure(SourceFile:41)
	at androidx.compose.runtime.Recomposer$effectJob$1$1.invoke(SourceFile:1536)
	at androidx.compose.foundation.gestures.DefaultScrollableState$scrollScope$1.scrollBy(SourceFile:17)
	at androidx.compose.runtime.RecomposeScopeImpl$end$1$2.invoke(SourceFile:1243)
	at androidx.compose.foundation.AndroidEdgeEffectOverscrollEffect.applyToScroll-Rhakbz0(SourceFile:358)
	at androidx.compose.foundation.gestures.ScrollingLogic.dispatchScroll-3eAAhYA(SourceFile:48)
	at androidx.compose.runtime.Latch$await$2$2.invoke(SourceFile:1020)
	at androidx.compose.foundation.gestures.ScrollingLogic$doFlingAnimation$2$scope$1.scrollBy(SourceFile:14)
	at androidx.navigation.NavController$navigate$5.invoke(SourceFile:513)
	at androidx.compose.animation.core.AnimatableKt.doAnimationFrameWithScale(SourceFile:55)
	at androidx.compose.animation.core.SuspendAnimationKt$animate$9.invoke(SourceFile:25)
	at com.machiav3lli.fdroid.utility.UtilsKt$findSuggestedProduct$2.invoke(SourceFile:145)
	at androidx.compose.runtime.BroadcastFrameClock.sendFrame(SourceFile:34)
	at com.machiav3lli.fdroid.pages.AppSheetKt$AppSheet$onActionClick$1.invoke(SourceFile:43)
	at androidx.compose.ui.platform.AndroidUiFrameClock$withFrameNanos$2$callback$1.doFrame(Unknown Source:6)
	at androidx.compose.ui.platform.AndroidUiDispatcher$dispatchCallback$1.doFrame(SourceFile:48)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1404)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1415)
	at android.view.Choreographer.doCallbacks(Choreographer.java:1015)
	at android.view.Choreographer.doFrame(Choreographer.java:941)
	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:1389)
	at android.os.Handler.handleCallback(Handler.java:959)
	at android.os.Handler.dispatchMessage(Handler.java:100)
	at android.os.Looper.loopOnce(Looper.java:232)
	at android.os.Looper.loop(Looper.java:317)
	at android.app.ActivityThread.main(ActivityThread.java:8592)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:580)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:878)
	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [androidx.compose.ui.platform.MotionDurationScaleImpl@f524521, androidx.compose.runtime.BroadcastFrameClock@f9dad46, StandaloneCoroutine{Cancelling}@b124b07, AndroidUiDispatcher@8104434]
```

**Comment by machiav3lli** [comment]
Actually https://github.com/NeoApplications/Neo-Store/commit/c8768f31a6ae55c1d0a9ebc8c9f6320e4dc7444c fixes this and not the other mentioned issue.

**Comment by machiav3lli** [comment]
This one is duplicate of #627

**Comment by machiav3lli** [comment]
@opusforlife2 The commit mentions #626, but that's not the issue that got fixed rather #627
